### DISCUSSION TITLE
Method to confirm production deployments with manual intervention

### DISCUSSION THREAD

**Discussion by ldileonardo** [vercel/community#3971] [Idea]
Hi,

We use turbo-repo as a mono-repo and we have code that deploys to another third party platform in parallel to a commit into a given branch.
Ideally for production we have some way to have it pause and require a manual confirmation prior to deployment so we can review the results of the parallel deployments first before letting the production deployment through.

Thanks.

**Comment by amyegan** [comment]
Hey @ldileonardo. You can do this with GitHub Actions or other workflow tools. This guide has a general outline with links to more specific instructions for GitHub Actions and Bitbucket Pipelines: https://vercel.com/guides/using-vercel-cli-for-custom-workflows

You can set [Ignored Build Step](https://vercel.com/docs/projects/overview#ignored-build-step) to `Don't build anything` if you want to disable automatic deployment and rely entirely on your custom workflow. Or you can set it to `Only build preview` if you want to keep automatic preview deployments and only use your custom workflow for production.

Alternatively, you can use [`git.deploymentEnabled`](https://vercel.com/docs/projects/project-configuration/git-configuration#git.deploymentenabled) in a `vercel.json` file to disable automatic deployment for individual branches or for the whole project.
